packages/webapp/src/pages/Connection/List.tsx

```diff
@@ -20,59 +20,55 @@ import { useStore } from '../../store';
 
 const defaultFilter = ['all'];
 
+function truncateWithEllipsis(text: string) {
+    if (text.length <= 6) {
+        return text;
+    }
+    return text.slice(0, 3) + '...' + text.slice(-3);
+}
+
 export default function ConnectionList() {
     const navigate = useNavigate();
     const env = useStore((state) => state.env);
     const { data, error, errorNotifications } = useConnections(env);
 
-    const [connections, setConnections] = useState<Connection[] | null>(null);
-    const [filteredConnections, setFilteredConnections] = useState<Connection[]>([]);
-    const [setNumberofErroredConnections, setNumberOfErroredConnections] = useState<number>(0);
-    const [selectedIntegration, setSelectedIntegration] = useState<string[]>(defaultFilter);
+    const connections: Connection[] = data?.connections || [];
+
+    const [selectedIntegrations, setSelectedIntegrations] = useState<string[]>(defaultFilter);
     const [connectionSearch, setConnectionSearch] = useState<string>('');
-    const [states, setStates] = useState<string[]>(defaultFilter);
+    const [selectedStatuses, setSelectedStatuses] = useState<string[]>(defaultFilter);
+    const [selectedCustomerDomains, setSelectedCustomerDomains] = useState<string[]>(defaultFilter);
 
-    useEffect(() => {
```

Comment:
This useEffect hook was entirely duplicative of the second one from what I could tell